spec/spec_helper.rb

```diff
@@ -4,6 +4,7 @@
 end
 
 require "webmock/rspec"
+require "climate_control"
```

Comment:
Hm, I guess the same could be said about `webmock/rspec` above as well. If so, then I probably won't need to do `require "spec_helper"` in the spec itself at all, but it's be missing the `WebMock` config in there. Thoughts?